typing/printtyp.ml

```diff
@@ -332,10 +332,12 @@ let ident ppf id = pp_print_string ppf
 (* Print a path *)
 
 let ident_stdlib = Ident.create_persistent "Stdlib"
+let opened_idents = lazy (ident_stdlib :: List.map Ident.create_persistent !Clflags.open_modules)
+let is_opened id = List.exists (Ident.same id) (Lazy.force opened_idents)
 
 let non_shadowed_pervasive = function
```

Comment:
The name seems even less fitting with this change, `from_initially_open_modules` might fit better?